packages/assets-controllers/src/NftController.ts

```diff
@@ -52,6 +56,77 @@ type SuggestedNftMeta = {
   origin: string;
 };
 
+export enum OpenSeaV2ChainIds {
+  ethereum = 'ethereum',
+}
+
+export type OpenSeaV2GetNftResponse = { nft: OpenSeaV2DetailedNft };
+
+export type OpenSeaV2Nft = {
+  identifier: string;
+  collection: string;
+  contract: string;
+  token_standard: string;
+  name: string;
+  description: string;
+  image_url: string;
+  metadata_url: string;
```

Comment:
More optional fields
```suggestion
  image_url?: string;
  metadata_url?: string;
```